src/content/docs/en/installation/rpi.mdx

```diff
@@ -33,11 +33,18 @@ The best experience will be with a SATA SSD connected over an adapter. A USB 3.0
   it on another device.
 </Alert>
 
+First, you will need to decompress it.
+
+- On Linux, run `unzstd *FILENAME.img.zst*`, you may need to install the `zstd` package
+- On Windows 11 or ChromeOS, zstd is natively supported. Decompress like you would a .zip
+- On Windows 10, use 7zip
+- On macOS, install `zstd` from homebrew, then run the above command for Linux
```

Comment:
```suggestion
- On macOS, install `zstd` from Homebrew, then run the above command for Linux
```
also maybe just say the command again lol, then you can include the install command

Reply:
Good idea